### DISCUSSION TITLE
Mettre à disposition des données sur data.gouv.fr

### DISCUSSION THREAD

**Discussion by alexisig** [MTES-MCT/sparte#437] [Idées]
@Phil-Loriot @inesdartiguenave j'ai récupéré les accès de notre espace sur data.gouv :
https://www.data.gouv.fr/fr/organizations/mon-diagnostic-artificialisation/
je peux vous partager les accès si vous avez un compte dessus (j'ai juste besoin de votre mail)

ça pourrait être une bonne option proposer en téléchargement nos données les plus lourdes, sans impacter notre serveur (couches artifs, imperméabilisation etc.)

**Comment by inesdartiguenave** [comment]
J'ai pas de compte dessus mais très bonne idée de se positionner sur data.gouv

________________________________
De : Alexis ***@***.***>
Envoyé : jeudi 13 juin 2024 14:02:05
À : MTES-MCT/sparte
Cc : ines.dartiguenave; Mention
Objet : [MTES-MCT/sparte] Mettre à disposition des données sur data.gouv.fr (Discussion #437)


@Phil-Loriot<https://github.com/Phil-Loriot> @inesdartiguenave<https://github.com/inesdartiguenave> j'ai récupéré les accès de notre espace sur data.gouv :
https://www.data.gouv.fr/fr/organizations/mon-diagnostic-artificialisation/
je peux vous partager les accès si vous avez un compte dessus (j'ai juste besoin de votre mail)

ça pourrait être une bonne option proposer en téléchargement nos données les plus lourdes, sans impacter notre serveur (couches artifs, imperméabilisation etc.)

**Comment by Phil-Loriot** [comment]
moi non plus, pas de compte... mais très bonne idée !

**Comment by alexisig** [comment]
J'ai remis les infos de notre fiche produit beta pour la description, et mis en ligne les couches d'artif : 
https://www.data.gouv.fr/fr/organizations/mon-diagnostic-artificialisation/
